packages/react-ui-kit/src/Icon/PlaneIcon.jsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Wire
+ * Copyright (C) 2017 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import IconBase from './IconBase';
+import React from 'react';
+
+class PlaneIcon extends IconBase {
```

Comment:
Let's stick to the names the design team uses: https://github.com/wireapp/wire-webapp/blob/dev/app/style/fonts/zeta-neue.css